### ISSUE TITLE
DSGVO problem (Google Fonts)

### ISSUE THREAD

**Issue by Minecodes** [Hexnrik/Developer-Website-Template#1]
This implementation of Google Fonts should be changed because this can lead to problems with law firms and the [DSGVO](https://www.heise.de/news/Neue-Abmahnwelle-Wieder-gehen-Schreiben-wegen-Google-Fonts-raus-7322064.html).
Please implement it with locally saved fonts. For this, you can use [Google Webfonts Helper](https://gwfh.mranftl.com/fonts)

**Comment by Hexnrik** [comment]
Yes, I know, but I'm too busy right now, the latest websites that are set to private by me already have all that, I just haven't gotten around to updating them yet.

**Comment by Hexnrik** [comment]
**Hey @Minecodes ,**

I'll get to the problem in the course of the week, maybe. I'll have a look at the new code for this website in the course of the week.

**Comment by Hexnrik** [comment]
So I've put it on my ToDo, I'll see when I get around to it.
